[Bug] Maven build faalt: com.sleepycat:je dependency niet resolvable

## Beschrijving

De Maven build faalt in CI (en mogelijk lokaal) door een onoplosbare dependency.

## Foutmelding

```
[ERROR] Failed to read artifact descriptor for com.sleepycat:je:jar:7.0.6
[ERROR] Caused by: Could not transfer artifact com.sleepycat:je:pom:7.0.6
  from/to netbeans (https://netbeans.apidesign.org/maven2/):
  PKIX path validation failed: java.security.cert.CertPathValidatorException: validity check failed
```

## Analyse

- `pom.xml` definieert `com.sleepycat:je:5.0.73`, maar een transitieve dependency trekt versie `7.0.6` binnen
- Het `netbeans` repository (`https://netbeans.apidesign.org/maven2/`) heeft een verlopen SSL-certificaat
- Dit is een upstream issue in de originele uniCenta oPOS broncode

## Mogelijke oplossingen

1. **Repository verwijderen/vervangen**: het `netbeans` repository uit `pom.xml` verwijderen als het niet noodzakelijk is
2. **Dependency versie pinnen**: expliciet `com.sleepycat:je:5.0.73` forceren en de transitieve dependency excluden
3. **Alternatief repository**: Berkeley DB JE is beschikbaar via Maven Central voor oudere versies

## Reproduceren

```bash
mvn clean package
```

## Omgeving

- Java 11 (Temurin)
- Maven (GitHub Actions runner: ubuntu-latest)

--- a/pom.xml
+++ b/pom.xml
@@ -564,18 +564,8 @@
       <url>https://repo1.maven.org/maven2/</url>
       <layout>default</layout>
     </repository>
-    <!--These are needed for Swing/Netbeans -->
-    <repository>
-      <id>netbeans</id>
-      <name>NetBeans</name>
-      <url>https://netbeans.apidesign.org/maven2/</url>
-    </repository>
-    <!--These are needed for Swing/Netbeans -->
-    <repository>
-      <id>flatlaf</id>
-      <name>flatlaf</name>
-      <url>https://dev.webswing.org/public/nexus/repository/webswing-3rd-parties//</url>
-    </repository>
+    <!-- netbeans repo verwijderd: SSL-certificaat verlopen (PKIX validation failure) -->
+    <!-- flatlaf repo verwijderd: niet nodig, flatlaf is beschikbaar via Maven Central -->
   </repositories>
   <distributionManagement>
     <repository>

